tests/test_geospatial.py

```diff
@@ -0,0 +1,418 @@
+import glob
+import json
+import os
+import shutil
+import tempfile
+import unittest
+import warnings
+from pathlib import Path
+from unittest import mock
+
+import affine
+import fiona
+import numpy as np
+import rasterio
+import shapely
```

Comment:
You'll need to move the imports of these into only the methods where they are directly used, to avoid having them at the module level.